remoteappmanager/file_config.py

```diff
@@ -77,18 +70,18 @@ def __init__(self, *args, **kwargs):
             self.docker_host = "unix://var/run/docker.sock"
 
         self.tls_verify = (env.get("DOCKER_TLS_VERIFY", "") != "")
```

Comment:
Nevertheless, the referenced issue on docker and its referenced issues show that I am not the only one surprised by this ;)